proposals/aggregate.json

```diff
@@ -0,0 +1,135 @@
+{
+    "id": "aggregate",
+    "summary": "Aggregation based on general intervals",
+    "description": "Computes a aggregation based on an array of intervals.\n\nThe computed values will be projected to the labels. If no labels are specified, the lower value of the interval will be used as label for the corresponding values. In case of a conflict (i.e. the user-specified values for the lower values of the intervals are not distinct), the user-defined labels must be specified in the parameter `labels` as otherwise a `DistinctDimensionLabelsRequired` exception would be thrown. The number of user-defined labels and the number of intervals need to be equal.",
```

Comment:
> The computed values will be projected to the labels

what does "projecting" to a label mean?

Reply:
"mapped" is probably better than "projected". Maybe it can also just be removed.

(I should also check whether this wording also exists in aggregate_temporal...)